### ISSUE TITLE
[BUG/ISSUE] BXHEIGHT is included twice when computing AOD from ISOP SOA diagnostic

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#208]
It was noted that the AOD from isoprene SOA diagnostic in the 12.7.0 benchmark was much higher than it should have been:

![image](https://user-images.githubusercontent.com/5880296/73291609-804fc000-41ce-11ea-9476-1500974280d0.png)

After consultation with Eloise Marais, we discovered that there was an extra BXHEIGHT variable in the equation for ISOPOD in aerosol_mod.F:

```Fortran
               ! Get the AOD contribution from isoprene SOA only (eam, 2014)
               IF ( N == 3 .and. Is_ComplexSOA ) THEN
                  ISOPOD(I,J,L,IWV) = SCALEOD*BXHEIGHT(I,J,L)*0.75d0
     &                              * State_Met%BXHEIGHT(I,J,L)
     &                              * ISOAAQ(I,J,L) * QQAA(IWV,1,N) /
     &                              ( MSDENS(N) * REAA(1,N) * 1.0D-6 )
               ENDIF
```

After removing the extraneous reference to ``State_Met%BXHEIGHT``` we recomputed the diagnostic, which seems to have moved it to the proper order of magnitude.

![image](https://user-images.githubusercontent.com/5880296/73291845-05d37000-41cf-11ea-890c-584892cfa6e7.png)

We are still trying to determine if there the AOD from ISOP SOA is double-counted in the total AOD diagnostic.

**Comment by FeiYao-Edinburgh** [comment]
Thanks for your report! I have also found this bug in GC 12.5.0. ISOPOD's precursor, ISOAAQ, also goes to OCPISOA, which further goes to WAERSL(I,J,L,3) and finally AODHyg550nm_POA1 taking wavelength=550nm as an example. I checked these calculations and think everything should be fine.



> total AOD diagnostic.

Do you have total diagnostic? Or do you mean diagnostic like AODHyg550nm_POA1?

**Comment by yantosca** [comment]
The total AOD is computed in the python code that plots the benchmark results.  So we sum them up offline.

**Comment by msulprizio** [comment]
I checked the code in aerosol_mod.F and believe that isoprene SOA is not double counted there as @FeiYao-Edinburgh also found. 

However, it does appear to be double counted in `benchmark.py` (found in https://github.com/geoschem/gcpy) where `AODTotal` is computed by summing all AOD diagnostics, including both OCPI (which includes isoprene SOA) and isoprene SOA. This is a bug in GCPy and has been reported as an issue there (see https://github.com/geoschem/gcpy/issues/65).

I will close this issue to reflect that the diagnostic fix to remove including BXHEIGHT twice in computing AOD from isoprene SOA has been fixed in 12.7.1.
